CMakeLists.txt

```diff
@@ -807,7 +807,7 @@ else()
 
   # Add default external projects.
   iree_llvm_add_external_project(mlir-iree-dialects ${CMAKE_CURRENT_SOURCE_DIR}/llvm-external-projects/iree-dialects)
-  iree_llvm_add_external_project(mlir-hlo ${CMAKE_CURRENT_SOURCE_DIR}/third_party/mlir-hlo)
+  iree_llvm_add_external_project(stablehlo ${CMAKE_CURRENT_SOURCE_DIR}/third_party/stablehlo)
   if(IREE_INPUT_TORCH)
     iree_llvm_add_external_project(torch-mlir-dialects ${CMAKE_CURRENT_SOURCE_DIR}/third_party/torch-mlir-dialects)
   endif()
```

Comment:
Keeping this switch simple for now is fine, but we might want to put `stablehlo` behind `IREE_INPUT_STABLEHLO` like how `torch-mlir-dialects` is behind `IREE_INPUT_TORCH`

Reply:
Yeah, I want to avoid changing too many things at one